app/services/get_subject_name.rb

```diff
@@ -0,0 +1,25 @@
+module GetSubjectName
+  SUBJECT_SYNONYMS = {
+    'Art': 'Art and design',
+    'Design Technology': 'Design and technology',
+    'English Literature': 'English',
+    'English Language': 'English',
+    'Health and Social care': 'Health and social care',
+    'Latin': 'Classics',
+    'Maths': 'Mathematics',
+    'Media Studies': 'Media studies',
+    'Physical Education': 'Physical education',
+    'Religious Studies': 'Religious education',
+    'General Science': 'Science'
+  }.stringify_keys
+
+  def get_subject_name(subject)
+    return nil if subject.blank?
+
+    subject_option_names = SUBJECT_OPTIONS.map { |subject_option| subject_option.first }
```

Comment:
Here you can use:

`SUBJECT_OPTIONS.map(&:first)`

https://borgs.cybrilla.com/tils/symbol-to-proc-in-ruby/